src/index.html

```diff
@@ -17,16 +17,43 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search"
+      data-qa="big"
+    >
+      <button class="search--button">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class name 'search--button' seems to indicate a BEM modifier, but it is not clear what it is modifying.